recipe/meta.yaml

```
@@ -12,41 +12,50 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - fonttools = fontTools.__main__:main
     - ttx = fontTools.ttx:main
     - pyftsubset = fontTools.subset:main
     - pyftmerge = fontTools.merge:main
+  script_env:
+    - FONTTOOLS_WITH_CYTHON=1
 
 requirements:
+  build:
+    - {{ compiler('c') }}
   host:
-    - python >=3.6
+    - python
+    - cython
     - setuptools
     - pip
     - wheel
   run:
-    - python >=3.6
-    # for fontTools.sfnt and fontTools.woff2: to compress/uncompress
-    # WOFF 1.0 and WOFF 2.0 webfonts.
+    - python
     - brotli >=1.0.1
-    # 5/17/2021: the unicodedata2 extension module doesn't work on PyPy.
-    # Python 3.9 already has Unicode 13.0, so the backport is not needed.
-    - unicodedata2 >=13.0.0  # [py<39 and (python_impl != "pypy")]
-    - munkres
+    - unicodedata2 >=15.1.0 # [py<313]
+  run_constrained:
+    - lxml >=4.0
```

```suggestion
    - fs >=2.2.0,< 3
    - lxml >=4.0
    - zopfli >=0.1.4
```
https://github.com/fonttools/fonttools/blob/4.51.0/setup.py#L93C10-L93C26
https://github.com/fonttools/fonttools/blob/4.51.0/setup.py#L105

---

And don't forget to remove the staging channel before merge!

---

Added `fs`, and once approved I'll remove the staging channels. Thank you for the reminder!